docs/testing/testing-craft/getting-started.md

```diff
@@ -108,6 +108,9 @@ define('CRAFT_TRANSLATIONS_PATH', __DIR__ . '/_craft/translations');
 define('CRAFT_VENDOR_PATH', dirname(__DIR__).'/vendor');
 
 TestSetup::configureCraft();
+
+// Prevent `headers already sent` error when running tests in PhpStorm
+ob_start ();
```

Comment:
Any reason we don't put this in `craft\test\TestSetup::configureCraft()`?

Reply:
You're right, I've moved it into `TestSetup`.